include/linux/hdmi.h

```diff
@@ -156,7 +156,7 @@ enum hdmi_content_type {
 };
 
 enum hdmi_metadata_type {
-	HDMI_STATIC_METADATA_TYPE1 = 1,
+	HDMI_STATIC_METADATA_TYPE1 = 0,
```

Comment:
Yes this looks like it is confusion between defining the bit masking and how to define it in the headers as an absolute value or bit position.

6 of one, half a dozen of the other between changing the header vs amending ```hdr_metadata_type```
```
static uint8_t hdr_metadata_type(const u8 *edid_ext)
{
	return edid_ext[3] &
		BIT(HDMI_STATIC_METADATA_TYPE1);
}
```
Has this been passed upstream at all?

Reply:
Yes see https://lists.freedesktop.org/archives/dri-devel/2021-January/295187.html and https://github.com/xbmc/xbmc/issues/19122

> Oh, my bad, I thought this was in drm_mode.h, but it's not. Then yeah should be completely fine to fix it.

Reply:
That thread's familiar now - I must have read it in passsing.
Still not merged (https://patchwork.freedesktop.org/patch/415578/?series=86212&rev=1), but it looks like it's largely been accepted as required.